flexmeasures/data/schemas/account.py

```diff
@@ -35,6 +36,21 @@ class Meta:
     account_roles = fields.Nested("AccountRoleSchema", exclude=("accounts",), many=True)
     consultancy_account_id = ma.auto_field()
 
+    @validates("primary_color")
+    def validate_primary_color(self, value):
+        if value is not None and not validate_color_hex(None, "primary_color", value):
+            raise ValidationError("Invalid HEX color for primary_color.")
+
+    @validates("secondary_color")
+    def validate_secondary_color(self, value):
+        if value is not None and not validate_color_hex(None, "secondary_color", value):
+            raise ValidationError("Invalid HEX color for secondary_color.")
+
+    @validates("logo_url")
```

Comment:
This actually worked: I saw "'ccc' is not a valid URL. " when I tried that.

Reply:
I just now understand what you meant here. You are pointing that the value is being displayed in the error message right?